examples/permsapp.py

```diff
@@ -56,6 +57,7 @@
 from __future__ import absolute_import, print_function
 
 from app import app, fixtures, rec1_uuid
+from flask_menu import Menu
```

Comment:
Without these two lines I get, when running `$ FLASK_APP=examples/permsapp.py flask run --debugger` and navigating to `/records/1`:

```python
[2016-07-15 13:28:59,415] ERROR in app: Exception on /records/1 [GET]
Traceback (most recent call last):
  File "/Users/jacquerie/.virtualenvs/access/lib/python2.7/site-packages/flask/app.py", line 1988, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/jacquerie/.virtualenvs/access/lib/python2.7/site-packages/flask/app.py", line 1634, in full_dispatch_request
    self.try_trigger_before_first_request_functions()
  File "/Users/jacquerie/.virtualenvs/access/lib/python2.7/site-packages/flask/app.py", line 1660, in try_trigger_before_first_request_functions
    func()
  File "/Users/jacquerie/.virtualenvs/access/lib/python2.7/site-packages/invenio_accounts/views.py", line 67, in init_menu
    item = current_menu.submenu('breadcrumbs.settings')
  File "/Users/jacquerie/.virtualenvs/access/lib/python2.7/site-packages/werkzeug/local.py", line 343, in __getattr__
    return getattr(self._get_current_object(), name)
  File "/Users/jacquerie/.virtualenvs/access/lib/python2.7/site-packages/werkzeug/local.py", line 302, in _get_current_object
    return self.__local()
  File "/Users/jacquerie/.virtualenvs/access/lib/python2.7/site-packages/flask_menu/__init__.py", line 65, in root
    return current_app.extensions['menu']
KeyError: 'menu'
127.0.0.1 - - [15/Jul/2016 13:28:59] "GET /records/1 HTTP/1.1" 500 -
```